script/options.js

```diff
@@ -34,8 +39,11 @@ function gatherFromCommandLine() {
 }
 
 function applyDefaultOptions(options) {
+  const contentRoot = '../content';
+
   Object.assign(options, {
-    contentRoot: '../content',
+    contentRoot,
+    contentPagesRoot: `${contentRoot}/pages`,
```

Comment:
I added  a `contentPagesRoot` property that points at the standard Vets.gov content directory, but is overridden in the step for defining Brand Consolidation options to look at the `content-directory` build argument.